tests/testsuite/build_script.rs

```diff
@@ -149,6 +151,46 @@ fn custom_build_env_vars() {
         p.cargo("build").arg("--features").arg("bar_feat"),
         execs().with_status(0),
     );
+
+    // Test passing linker from .cargo/config to env var
+    if cross_compile::disabled() { return; }
+    let target = cross_compile::alternate();
```

Comment:
I think it may be possible to not gate this test on cross compiling as well, if you don't pass `--target` then the env var for the host should still be set I think?

Reply:
Ah right yes indeed! I think this is fine to stay in that case